models/cube.js

```diff
@@ -1,5 +1,7 @@
 const mongoose = require('mongoose');
 
+const CURRENT_SCHEMA_VERSION = require('./migrations/deckMigrations').slice(-1)[0].version;
```

Comment:
importing and then slice, and then taking index? there is no good reason why we should be doing this. Migrations should be a one shot, disposable script. No need to make these patterns that are nonsensical if you aren't familiar with the process

Reply:
I can make it more intuitive, but I'd like a system for tracking the schema versions of documents.